apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/oversendelsebrev/OversendelseBrevService.kt

```diff
@@ -130,7 +130,7 @@ class OversendelseBrevServiceImpl(
             )
         val verge =
             grunnlagService.hentVergeForSak(
-                sakType = behandlingService.hentSak(klage.sak.id, brukerTokenInfo).sakType,
+                sakType = klage.sak.sakType,
```

Comment:
@oyvindsh, vet du om det er en grunn til at `sakType` ble hentet fra behandling, i stedet for rett fra sak-objektet i klagen? Kan det være diff der, eller er dette bare gammel moro som aldri ble ryddet i?

Reply:
Bare gammel moro, her burde saktypen i klagen bare brukes ja